src/client/src/tscServer.c

```diff
@@ -1534,6 +1534,13 @@ int tscBuildCreateTableMsg(SSqlObj *pSql, SSqlInfo *pInfo) {
       pMsg += sizeof(SCreateTableMsg);
 
       SCreatedTableInfo* p = taosArrayGet(list, i);
+      //what pCreate->tableName point is a fixed char array which size is 237
+      //what p->fullname point is a char*
+      //before the time we copy p->fullname to pCreate->tableName , we need to check the length of p->fullname
+      while (strlen(p->fullname) > 237) {
+          printf("length of p->fullname should not exced 237\n");
```

Comment:
Print out the error message is not a good idea here, maybe use tscError  instead is a better idea?